source/images/step/latest.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<circle cx="12" cy="12" r="10" fill="#0FC389"/>
+<path d="M7.57666 12.2071L9.06365 10.7929L10.9488 12.5858L14.7191 9L16.2061 10.4142L10.9488 15.4142L7.57666 12.2071Z" fill="white"/>
```

Comment:
Remove `fill` from these elements so we can
- specify the colors with our constants in the stylesheets
- reuse the same .svg files in other places, with different colors

Reply:
I'd leave it here since it provides a reasonable default in case we forget to specify it in CSS. A wrong color is better than a broken UI.